### ISSUE TITLE
Quadratic Formula Explanation

### ISSUE THREAD

**Issue by Kyle-P** [Kyle-P/Mathematics#8]
Please add an explanation to users for how the zeroes were computed based on the [quadratic formula program](https://github.com/Kyle-P/Mathematics/blob/master/Algebra/QuadraticFormula.java). 

Remember that each program must explain to users how answers are generated rather than just providing an answer.

**Comment by manimanasamylavarapu** [comment]
when i click on https://github.com/Kyle-Pu/Mathematics/blob/master/Algebra/QuadraticFormula.java
it says page not found

**Comment by manimanasamylavarapu** [comment]
can u please assign this to me

**Comment by Kyle-Pu** [comment]
Hi! The specific file for editing now lives at https://github.com/Kyle-Pu/Mathematics/blob/master/Algebra/Quadratic%20Formula/QuadraticFormulaExplanation.md. Thanks so much!

**Comment by manimanasamylavarapu** [comment]
So basically this has to be explained in more detail. Am I right :) >

**Comment by Kyle-Pu** [comment]
Yeah, just a little about how we actually did the calculations. Thank you :D
